### ISSUE TITLE
show data on map

### ISSUE THREAD

**Issue by vantreeseba** [codeforokc/school-finder#21]

**Comment by jvrousseau** [comment]
not sure if this is useful, but here's a dataset from ok.gov (statewide):

https://data.ok.gov/Education/Oklahoma-Institutions-of-Common-Education/y9pd-rjtd

cc: @joekarl @DevinClark

**Comment by joekarl** [comment]
Yeah saw that, trick will be xrefing the data we have with that data (probably have to go against school name which will be spotty at best)

**Comment by Mike-Loffland** [comment]
We're only concerned with Oklahoma City Schools... correct?

Filtering the school data at data.ok.gov to only include OKC schools returns 184 results. This is significantly lower than the 363 school entries in our current dataset.

**Comment by mkchandler** [comment]
We want to also include the surrounding suburbs of OKC if possible (Edmond, Mustang, Yukon, Bethany, etc.)

**Comment by Mike-Loffland** [comment]
@mkchandler gotcha... I'm just trying to figure out what geographic boundary was used to produce the dataset at http://data.okc.gov/applications/datadownload/forms/DownloadDetails.aspx?DataSetID=17.

**Comment by Mike-Loffland** [comment]
Here is a filtered view of the OKC Metro area.

https://data.ok.gov/Education/OKC-METRO/4whz-7phq

@joekarl I saw that you have a branch that is attempting to make the schools as points instead of polygons. The data on data.ok.gov has lat, long, and location columns. I assume that location would be the point we need?

**Comment by joekarl** [comment]
Yeah, those columns are the ones we'd need to cross reference with our existing polygon data from the city (should be able to see which feature from the geojson contains the school point from ok.data.gov).
